File: SustitucionMOAModel/Dto/LegajoExternoDto.cs
using System.Collections.Generic;

namespace SustitucionMOAModel.Dto
{
    public class LegajoExternoDto
    {        
        public List<LegajoDto> ListaLegajos { get; set; }
        public string NroOrdenDeCompra { get; set; }
        public string NroSolp { get; set; }
        public UsuarioDto Proveedor { get; set; }
        public string FechaAdjudicacionFormateado { get; set; }
    }
}

File: SustitucionMOA/src/app/modelos/legajoExternoDto.ts
import { LegajoDto } from "./compras/legajoDto"

export interface LegajoExternoDto {
    ListaLegajos?: LegajoDto[]
    OrdenDeCompraNro?: string
    FechaAdjudicacionFormateado?: string
    Proveedor?: any
}